README.md

```diff
@@ -4,16 +4,16 @@ Sopel module that fetches info about torrent links
 ## Current status
 
 The current support is very limited, just a proof of doing something. It fetches
-only from Nyaa, and then only a very few details. More information (and possibly
-other supported sites) will come later.
+only from Nyaa.si and Anidex.info, 
+and then only a very few details. More information (and possibly other supported sites) will come later.
```

Comment:
Too-short line and a too-long line, hmm.

Reply:
I have no idea how to break them.

Reply:
Just make each line roughly the same length as the others above. I think I wrapped it to 80 columns or so (maybe 72 if I was feeling silly).